test/widget/nav_bar/data/config_pair_test.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:xayn_design/src/widget/nav_bar/data/config_pair.dart';
+import 'package:xayn_design/src/widget/nav_bar/data/nav_bar_config.dart';
+import 'package:xayn_design/src/widget/nav_bar/nav_bar.dart';
+import 'package:xayn_design/src/widget/nav_bar/widget/nav_bar.dart';
+
+void main() {
+  test(
+    'GIVEN configPair THEN verify it extends Equatable',
+    () {
+      const pair = ConfigPair(FakeConfigUpdater(), []);
```

Comment:
Just checking, will we add documentation/comments to classes like `ConfigPair` in this story?

Reply:
I see you added documentation PR.. resolving this